packages/manager/tools/component-rollup-config/.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint'],
```

Comment:
😮

Reply:
@marie-j Could you please be more precise about your reaction?

Thanks!

Reply:
Why conversation has been resolved?

Meaning any preferences related to: https://github.com/palantir/tslint?